modules/statisticsInterface.js

```diff
@@ -0,0 +1,237 @@
+            .then(() => { setSettings() })
+
+
+    }
+    //добавляет в открытое поле статистики необходимые кнопки
+    function showSettingButton() {
+        let now = new Date();
+        let addStatInterface = document.createElement("div")
+        let addSettingsButton = document.createElement("div")
+        addStatInterface.innerHTML = `
+      <input type="month" id="month" name="month" min="${now.toISOString().substr(0, 4)}-01" value="${now.toISOString().substr(0, 7)}">
+        <input type="number" placeholder="Прошлый месяц" id="lastMonth">
+        <input list="sostavNumber" id="sostavNumber">
+        <datalist id="sostavNumber">
+        <option value="1">1</option><option value="2">2</option><option value="3">3</option><option value="4">4</option><option value="5">5</option><option value="6">6</option><option value="7">7</option><option value="8">8</option><option value="9">9</option><option value="10">10</option><option value="11">11</option><option value="12">12</option><option value="13">13</option><option value="14">14</option><option value="15">15</option><option value="16">16</option><option value="17">17</option><option value="18">18</option><option value="19">19</option><option value="20">20</option>
+        </datalist>
+        <div id="statStart" class="tampleteButton" style="height:23px">Посчитать</div>
+    `
+        let classBlok = document.createElement("style")
+        classBlok.innerHTML = `
+    body {
+      margin: 0;
+      padding: 0;
+      background: #f3f3f3;
+    }
+    .settings {
+      position: absolute;
+      left: 82%;
+      top: 40px;
+      width: 250px;
+      border: 1px solid rgb(205 205 224);
+      padding: 5px;
+      color: rgb(255 255 255);
+      background: rgb(50 50 51);
+      display: block;
+    }
+    #word_opts {
+      position: relative;
+      padding: 4px;
+      width: 60px;
+      height: 60px;
+      background: #c6c6c6;
+      outline: none;
+      border-radius: 7px;
+      box-shadow: inset 0 0 5px rgba(0,0,0, .2);
+      transition: 0.5s;
+    }
+
+    label input[type="checkbox"]:checked + #word_opts {
+      background: #02a9f4;
+    }
+
+    input[type="checkbox"] {
+      width: 40px;
+      height: 20px;
+      -webkit-appearance: none;
+      -moz-appearance: none;
+      background: #c6c6c6;
+      outline: none;
+      border-radius: 50px;
+      box-shadow: inset 0 0 5px rgba(0,0,0, .2);
+      transition: 0.5s;
+      position: absolute;
+      right: 40px;
+    }
+    input:checked[type="checkbox"] {
+      background: #02a9f4;
+    }
+    input[type="checkbox"]::before {
+      content: '';
+      position: absolute;
+      width: 20px;
+      height: 20px;
+      border-radius: 50%;
+      top: 0;
+      left: 0;
+      background: #fff;
+      transform: scale(1.1);
+      box-shadow: 0 2px 5px rgba(0,0,0, .2);
+      transition: 0.5s;
+    }
+    input:checked[type="checkbox"]::before {
+      left: 20px;
+    }
+    `
```

Comment:
you need to extract all the css to the css module :)

Reply:
ok. I will be try